app/main.py

```diff
@@ -1,2 +1,55 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.hash_table: list = [None] * 8
+        self.load_factor = 0.75
+
+    def __len__(self) -> int:
+        return self.length
+
+    def _hash(self, key: Any) -> int:
```

Comment:
```suggestion
    def _hash(self, key: Hashable) -> int:
```
please fix everywhere